### ISSUE TITLE
Green noise when using auto geometric distor. correction on high ISO

### ISSUE THREAD

**Issue by maboleth** [Beep6581/RawTherapee#5656]
Happens on high ISO RAW images, tested on Canon 5D Mark IV.

Having automatic geometric distortion correction for lens introduces some high-pitched green noise.

These are 100% crops of before and after turning this feature on/off. Please zoom in to see it for yourself.

![greendots-2](https://user-images.githubusercontent.com/2467520/74685375-aa454280-51ce-11ea-889b-6dd8b52ad90b.jpg)
![greendots-1](https://user-images.githubusercontent.com/2467520/74685379-aca79c80-51ce-11ea-82a3-69fc80bca19c.jpg)

**Comment by heckflosse** [comment]
Which version of RT?

**Comment by maboleth** [comment]
![test](https://user-images.githubusercontent.com/2467520/74730146-e9fb4100-5245-11ea-90d3-7f762bd3a944.jpg)
Latest 5.8.
It affects Geometric distortion only. Vignetting & Chromatic abr. are fine.

Affects ISOs from 12800 onwards, even though ISO 10000 still have them here and there, it's not as visible as on higher values. At ISO 25600 darker areas are so littered with it, the only way to make them disappear is to turn GD off.

I added another example of shadow area, they seem to appear in some sort of "wave" patterns (zoom in to see it).

**Comment by Beep6581** [comment]
@maboleth upload sample raw files.

**Comment by maboleth** [comment]
Okay, here it is.
http://cvrkutanje.net/_V3A9361.CR2

**Comment by Beep6581** [comment]
@maboleth pp3 too please.

**Comment by Thanatomanic** [comment]
@Beep6581 It is quite trivial to reproduce: apply Neutral, set Profiled lens correction to Automatic. Make sure ~~Distortion correction~~ Geometric distortion is ticked. The scaling of the pixels somehow amplifies the existing noise.

**Comment by maboleth** [comment]
> @Beep6581 It is quite trivial to reproduce: apply Neutral, set Profiled lens correction to Automatic. Make sure ~Distortion correction~ Geometric distortion is ticked. The scaling of the pixels somehow amplifies the existing noise.

Exactly. It's almost automatic. The thing I have noticed, the image (darker areas, shadows) seems 'peppered' with high pitched green noise in some kind of waves, lines, rarely it follows the existing noise.

Also when chroma noise is removed, this green noise remains. Nothing removes it except turning GD off.

**Comment by heckflosse** [comment]
Setting method to Linear it works fine. Seems Logarithmic does not work well on noisy images...

Left using Logarithmic, right using Linear
![grafik](https://user-images.githubusercontent.com/13733487/74748673-32782600-5269-11ea-992e-8fab91897666.png)

**Comment by maboleth** [comment]
^^ Correct. Logarithmic seems to cause the problem here.

Cranking the image contrast also makes this greatly visible. Also shows this pattern I mentioned before quite clearly.

![_V3A9343](https://user-images.githubusercontent.com/2467520/74749301-3c4e5900-526a-11ea-9928-10fc13524d71.jpg)

**Comment by maboleth** [comment]
Any news on this? Have you figured out the root cause?

**Comment by heckflosse** [comment]
Well logarithmic does not work well in this case. Linear works better in this case, though often logarithmic works better than linear. Maybe I should try the cube root approach of ART...

**Comment by maboleth** [comment]
Excellent, thank you! If you manage to tweak something, please let me now here, so I could test it. I'm using the official/stable 5.8 ver and not bleeding edge/beta.
